README.md

```diff
@@ -51,6 +51,14 @@ It will automatically start tracing.
 **This tool may also (temporarily) corrupt the state of your Xbox.**
 If this tool does not work, please retry a couple of times.
 
+### Contributing
+
+This project uses [Black](https://pypi.org/project/black/) for automatic formatting.
+You can use the pre-commit in the `githooks` directory to reformat any changes on commit
```

Comment:
I'm also skeptical about auto-formatting, although I'm fine with code conventions in some standard format (I'm not sure what `black` is or wether it's a standard for Python, but I'd assume that some good standard exists for Python, given how over-specified the language is).

Reply:
Not a fan, but other maintainers should decide

Reply:
I don't think having auto-formatting scripts in this repository is necessary, but I don't object strongly. I also have no objections to using black / following it's style conventions, etc.

Reply:
Fair enough; I'm still not sure I understand the reservations around automation, but the pre-commit hook is intentionally opt-in and I hope it'll save anybody who starts to contribute frequently from having to write the same thing manually.

Reply:
I don't dispute the value of automation nor code formatters, I use them regularly.

My argument is against having them run automatically before commit. When I commit something, I want it committed exactly as it is. Also have PTSD from automated tools mucking things up, so I want to be able to easily see and undo what the tool does.